README.md

```diff
@@ -6,14 +6,28 @@ applications within the GOV.UK stack.
 These are used to verify releases and also to provide Icinga alerts for major
 features.
 
+The tests are run against integration, staging and production environments, and are triggered by deployments of most GOV.UK applications, CDNs and associated dependencies (`grep -i "smokey" modules/govuk_jenkins/templates/jobs/*` in the [govuk-puppet repository](https://github.com/alphagov/govuk-puppet) for a full list).
+
 ## Technical documentation
 
 The smoke tests are based on [Cucumber](https://cucumber.io/). We use feature
 files to describe single applications (eg
 [`whitehall`](https://github.com/alphagov/whitehall),
 [`frontend`](https://github.com/alphagov/frontend)) or [cross-application behaviour](features/gov_uk.feature).
 
-### Running the tests
+### Installation
+
+Smokey requires Java to be installed, because of its [use of the BrowserMob Proxy](#use-of-browsermob-proxy). Note that if you're using the Development VM then Java is already installed.
+
+If you're not using the VM, [download the Java JDK from the OpenJDK website](https://openjdk.java.net/). Your download will look something like `openjdk-12.0.1_osx-x64_bin.tar.gz`. Untar this file and `sudo mv` it to `/Library/Java/JavaVirtualMachines`. You should now be able to run `java -version`.
```

Comment:
Given our [getting started instructions](https://docs.publishing.service.gov.uk/manual/get-started.html#1-install-some-dependencies) include installing rbenv using Homebrew, I'd suggest including the simple installation via that route. Something like this?

```suggestion
If you're not using the VM, or want to run Smokey on your host Mac, then you have two options:

* If you have Homebrew installed, you can install OpenJDK by using `brew cask install java`.
* If not, [download the Java JDK from the OpenJDK website](https://openjdk.java.net/), which will be a file named something like `openjdk-12.0.1_osx-x64_bin.tar.gz`. Untar this file (`tar zxf [FILE]`) and `sudo mv` it to `/Library/Java/JavaVirtualMachines`.

Once you've done this, run `java -version` to check the installation has been successful.
```

Reply:
Agreed, somehow I glossed over your homebrew suggestion - apologies!